Workbooks/MySQL Flexible Server/Auditing/Auditing.workbook

```diff
@@ -0,0 +1,225 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "**Administraive Actions on the service** (You need to send Azure Activity logs to Diagonistics Ref: https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings?WT.mc_id=Portal-Microsoft_Azure_Monitoring&tabs=CMD)",
+        "style": "info"
+      },
+      "name": "text - 2",
+      "styleSettings": {
+        "showBorder": true
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "AzureActivity\r\n| where CategoryValue == \"Administrative\"\r\n| where ActivityStatusValue == \"Success\" or ActivityStatusValue == \"Failed\"\r\n| project EventSubmissionTimestamp, substring(OperationNameValue,37),CallerIpAddress,Caller,ActivityStatusValue\r\n\r\n\r\n",
+        "size": 0,
+        "timeContext": {
+          "durationMs": 604800000
+        },
+        "queryType": 0,
+        "resourceType": "microsoft.dbformysql/flexibleservers",
+        "gridSettings": {
+          "sortBy": [
+            {
+              "itemKey": "EventSubmissionTimestamp",
+              "sortOrder": 1
+            }
+          ],
+          "labelSettings": [
+            {
+              "columnId": "EventSubmissionTimestamp",
+              "label": "Event TIme"
+            },
+            {
+              "columnId": "CallerIpAddress",
+              "label": "Caller IP"
+            },
+            {
+              "columnId": "ActivityStatusValue",
+              "label": "Status"
+            }
+          ]
+        },
+        "sortBy": [
+          {
+            "itemKey": "EventSubmissionTimestamp",
+            "sortOrder": 1
+          }
+        ]
+      },
+      "name": "query - 1"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "**Audit Summary** (You should enable MySQL Auditing Ref: https://docs.microsoft.com/en-us/azure/mysql/flexible-server/concepts-audit-logs)",
```

Comment:
instead of showing this everywhere all the time, you could possibly do queries to see if these things exist, and conditionally hide/show them?

Reply:
We fixed queries and added fwlink's for documentation,